content/blog/discover-the-power-of-data-center-monitoring-using-redfish-telemetry-and-cloud-native-tooling-part-2-streaming.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Discover the power of data center monitoring using Redfish telemetry and
+  cloud-native tooling: Part 2 - streaming"
+date: 2023-12-01T21:50:51.315Z
+author: Naveen Gupta
+authorimage: /img/photo-jpg-ng.jpg
+thumbnailimage: ""
+disable: false
+tags:
+  - ilo-restful-api
+  - Grafana
+  - Telegraf
+  - Telemetry
+  - InfluxDB
+  - Redfish
+  - monitoring
+---
+<style>
+li {
+   font-size: 27px;
+   line-height: 33px;
+   max-width: none;
+}
+</style>
+
+Unlock the full potential of data center monitoring with Redfish telemetry streaming and cutting-edge cloud-native tools. The second part of our data center monitoring whitepaper series delves into the world of streaming, showcasing the efficiency and scalability of HPE iLO's Redfish telemetry streaming using event subscription.
```

Comment:
You may want to make reference to your first blog post before talking about the second blog post.
Something like that below, but let's wait for Dale's editorial review before changing the introductory paragraph:

In the first part of the blog post series I discussed how coupling Redfish telemetry with cloud-native monitoring tools offers a robust solution for real-time monitoring, data analysis, visualization and alerting. The second part of our data center monitoring series delves int the world of streaming,...

Reply:
I concur with Denis. It would be very nice to remind folks that there's an article that came before this and make it easy for them to access. Especially if they hadn't noticed that you had written a previous article... providing them with a link back to it is a great way to expand the first article's readership.

Here's an alternate suggested beginning:
Ensuring the seamless operation of your data center is critical to business continuity. In my [first article in this series,](https://developer.hpe.com/blog/discover-the-power-of-data-center-monitoring-using-redfish-telemetry-and-cloud-native-tooling/)  I showed you how coupling Redfish telemetry, a standardized way to access telemetry data from servers and other devices, with cloud-native monitoring tools can improve the efficiency and insightfulness of your data center monitoring. In this second article, I'll show you how you can unlock the full potential of data center monitoring with Redfish telemetry streaming and cutting-edge cloud-native tools. Together, we'll delve deep into the world of streaming, as I showcase the efficiency and scalability of HPE iLO's Redfish telemetry streaming using event subscription.

(just to give you some insight into my suggestion for the rewrite - We can't assume that the reader has already read your previous article, so we need to try to entice them - once again - into reading your article. By making a statement up front regarding what this is all about, you set the context and help the reader to identify with the problem, recognizing that it's probably a problem he'd like to solve. And so he reads on...)